atuin-client/src/database.rs

```diff
@@ -515,7 +517,6 @@ impl Database for Sqlite {
     // but the time that the system marks it as deleted
     async fn delete(&self, mut h: History) -> Result<()> {
         let now = chrono::Utc::now();
-        h.command = String::from(""); // blank it
```

Comment:
I totally agree it shouldn't really be blank, thanks for highlighting this. However would you be able to set it to some random data instead? This way, `strings history.db` won't still show whatever the deleted value is